src/System.Buffers/tests/ArrayPool/UnitTests.cs

```diff
@@ -406,8 +406,12 @@ private static int RunWithListener(Action body, EventLevel level, Action<EventWr
                 int count = 0;
                 listener.RunWithCallback(e =>
                 {
-                    Interlocked.Increment(ref count);
-                    callback(e);
+                    // Don't bother tracking the GC polling event- it isn't deterministic
+                    if (e.EventId != 5)
```

Comment:
If the only place we raise this event is in BufferTrimPoll, maybe add a comment here saying we ignore the event raised when we check to trim buffers:
https://github.com/dotnet/corefx/blob/d4894ca021932067fc03578b510de9a3b6058276/src/Common/src/CoreLib/System/Buffers/ArrayPoolEventSource.cs#L100

Reply:
That is what the comment was trying to say.